README.md

```diff
@@ -53,9 +53,38 @@ This default may be changed in a future breaking release. In the meantime, manua
   (binding [org.httpkit.client/*default-client* sni-client/default-client]
     <...>)
 ```
-
 See `org.httpkit.client/*default-client*` docstring for more details.
 
+## Using Unix Domain Sockets in client and server
+
+Java native Unix Domain Sockets are available since Java 17, so to maintain backwards compatibility with JVMs prior to this,
```

Comment:
Just to double check: it looks like UDS support may have been available from Java 16 via [JEP-380](https://openjdk.org/jeps/380). Is it possible http-kit + UDS might work from Java >= 16?

Reply:
Assuming Java 16 for now, can fix README later if I've missed something 👍